crates/forge/src/lib.rs

```diff
@@ -462,35 +487,101 @@ fn run_with_fuzzing(
     };
     let mut fuzzer = RandomFuzzer::create(fuzzer_seed, fuzzer_runs, &args)?;
 
-    let mut results = vec![];
+    let mut tasks = FuturesUnordered::new();
```

Comment:
I wonder if we should use `FuturesOrdered` maybe to make fuzzing fail with the same args when using the same seed
cc @MaksymilianDemitraszek

Reply:
the reason why I used FuturesUnordered is faster :)